docs/setup.rst

```diff
@@ -140,16 +144,17 @@ should have a running Re:dash instance with everything you need to get
 started . Re:dash should be available using the server IP or DNS name
 you assigned to it. You can point your browser to this address, and login
 with the user "admin" (password: "admin"). But to make it useful, there are
-a few more steps that you need to manually do to complete the setup:
+a few manual steps to complete the setup:
 
-First ssh to your instance and change directory to ``/opt/redash``. If
-you're using the GCE image, switch to root (``sudo su``).
+First connect to your instance using ssh, and switch to root (``sudo su``).
+
+If you're using the Docker image, ``docker exec -i -t redash_redash_1 /bin/bash``, will put you in ``/opt/redash/current`` as root.
```

Comment:
1. They should use Compose's exec. This way no need to assume that the name will be `redash_redash_1`.
2. I'm not sure this is needed.  Is there a change they can'tdo via the env variables?